### PR TITLE
Compatable with newer gptq 

### PR DESCRIPTION
**Pull request by catalpaaa**
yea they are changing up shits again

### CODE REVIEW

**Comment by catalpaaa** [comment]
> Thanks. I don't have a triton installation to test, so I'll tag @sgsdxzy and @EyeDeck for review.
sure, they just moved modelutils into utils/
i just made it so it looks for that too

File: modules/GPTQ_loader.py [hunk 1]
```diff
@@ -12,13 +12,14 @@
...
 import llama_inference_offload
-from modelutils import find_layers
 
 try:
     from quant import make_quant
+    from modelutils import find_layers
     is_triton = False
 except ImportError:
     import quant
+    from utils.modelutils import find_layers
```
Comment (sgsdxzy) [inline modules/GPTQ_loader.py:22]: I think it is fine to just `from utils import find_layers`

**Comment by EyeDeck** [comment]
Works for me.

Without this patch, the latest GPTQ-for-LLaMA crashes with:
```
Traceback (most recent call last):
  File "/home/anon/text-generation-webui/server.py", line 912, in <module>
    shared.model, shared.tokenizer = load_model(shared.model_name)
  File "/home/anon/text-generation-webui/modules/models.py", line 125, in load_model
    from modules.GPTQ_loader import load_quantized
  File "/home/anon/text-generation-webui/modules/GPTQ_loader.py", line 15, in <module>
    from modelutils import find_layers
ModuleNotFoundError: No module named 'modelutils'
```
Maybe some day we'll stop needing to fight with random breaking changes every three days :')

**Comment by oobabooga** [comment]
I merged #1451 which had the simpler import syntax

**Comment by USBhost** [comment]
I did not see this pull mybad.

**Pull request closed without merging**